src/App.vue

```diff
@@ -19,7 +43,18 @@ statePersistorLayersService.bootstrapLayers()
       </div>
       <div class="grow text-center">search</div>
       <div>
-        <ul class="h-full">
+        <ul class="h-full flex">
+          <li>
+            <button
+              class="flex items-center before:font-icons before:text-3xl before:w-16 text-primary uppercase h-full mr-3"
```

Comment:
Click action will be implemented with `layersOpen` I guess.

Reply:
yes 😉